developmentseed/pearl-frontend
# Fix empty AOI list state

Fix #59. 

How to test:

- Open an existing project and delete all AOIs, the app should not crash
- Reload project with an empty AOI list, the UI should ask for the first AOI

@geohacker ready for review.
